source/whats-new.txt

```diff
@@ -90,7 +164,7 @@ New features of the 4.10 driver release include:
   retry operations. The driver now correctly emits one log message for
   each retry operation.
 - The ``org.bson.codecs.Parameterizable`` interface is deprecated. Instead of
-  implementing this interface on a custom ``Codec`` type, you should
+  implementing this interface on a custom ``Codec`` type,
```

Comment:
Nit: Missing a verb here now that we have removed "should"

Reply:
I think "override" is the verb, and I wasn't able to identify anything else missing. If I'm missing anything, could you explain? Thanks!